Assets/Scripts/BlueAnimaMovementBehaviour.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class BlueAnimaMovementBehaviour : MonoBehaviour
+{
+    Physics2D Physics2D;
+    // Start is called before the first frame update
+    void Start()
+    {
+        _addRigidBody();
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+
+    }
+
+    
+
+    void _addRigidBody()
+    {
+        this.gameObject.AddComponent<Rigidbody>();
+        var rb = this.GetComponent<Rigidbody2D>();
+
+        //重さの調整
+        rb.useAutoMass = true;
+        float massValue = rb.mass;
+        rb.useAutoMass = false;
+        rb.mass = massValue;

```

Comment:
```suggestion
        var rb = GetComponent<Rigidbody2D>();
        rb.useAutoMass = true;
```
僕が提案するのが遅かったですねすみません。UseAutoMassはデフォルトでdisabledなのでこのコードはいいと思います。ただ、質量計算に関しては今日説明したdensityを使ったほうがきれいに書けるかと思います。